### DISCUSSION TITLE
Would iOS ipa be common thing on GitHub releases?

### DISCUSSION THREAD

**Discussion by Spidy123222** [ppy/osu#22574] [Q&A] [answered]
I’m not sure where to ask this as it’s not a bug but a question on if the ipa release will be common on new versions as I noticed on latest it seems to have a ipa release which I haven’t seen in GitHub releases. 

My second question is will the ipa internal app version be changed on updates. I ask this because I run an AltStore app source and AltStore checks ipa version and the sources version number it’s supposed to be. It only queues an update if the source version is higher than the users current installed version. Atm the ipa seems to use 0.1.0 as the version and I use a program to update the version number based on ipa version it fetches from GitHub releases automatically using altstoreparser. Also tell me if this isn’t right area to ask this. Doesn’t seem to have a clear area to ask this other than maybe help discussion thread hints me posting here.

**Comment by peppy** [comment, marked as answer]
I intentionally added it as a trial, but do not count on it staying around.

Versioning is as intended. Main version will not be incremented until we start pushing app store releases, and will follow standard practices.

**Reply by Spidy123222** [reply]
Oh ok. It works really well, no issues at all from what I find. I don't have a mac and I find using TestFlight to be a problem if I want others to use the app but the TestFlight can be full. This kinda gives us a sense of peace we can keep the app on our device and share the experience. I wasn't sure osu!lazer would qualify for the AppStore as it doesn't allow external unchecked checked code but i could be totally wrong. But what made me think this is how some apps like Xbox Xcloud couldn't get added to the AppStore and the games they had on it would need to be reviewed for each one but in this case beatmaps. Hope things go well.

Thanks for the great program. Also used it on my windows pc.

The source I made is a public one called Sidestore community store (also open source on GitHub). Do you think it’s fine for me to have a download there since it’s your app. Just wondering. It just directly downloads from this GitHub with info about app. (Github: https://github.com/SideStore/Community-Source )

**Reply by peppy** [reply]
Do not publish it.
